docs/Concepts/4.md

```diff
@@ -1,60 +1,104 @@
+- Enough peers in the Waku Relay mesh (target is 6),
+- Enough peers in reserve, in case current peers are overloaded or go offline,
+- Peers with specific Waku capabilities (e.g. Store, Light Push, Filter).
 
-Waku v2 by default guarantees pseudonymity for all of the protocol layers since parties do not have to disclose their true identity
-and instead they utilize libp2p `PeerID` as their identifiers.
-While pseudonymity is an appealing security feature, it does not guarantee full anonymity since the actions taken under the same pseudonym
-i.e., `PeerID` can be linked together and potentially result in the re-identification of the true actor.
+## Default Bootstrap Mechanism
 
-### Anonymity / Unlinkability
+The default bootstrap mechanism is to connect to the Status' nim-waku prod fleet.
 
-At a high level, anonymity is the inability of an adversary in linking an actor to its data/performed action (the actor and action are context-dependent).
-To be precise about linkability, we use the term Personally Identifiable Information (PII) to refer to any piece of data that could potentially be used to uniquely identify a party.
-For example, the signature verification key, and the hash of one's static IP address are unique for each user and hence count as PII.
-Notice that users' actions can be traced through their PIIs (e.g., signatures) and hence result in their re-identification risk.
-As such, we seek anonymity by avoiding linkability between actions and the actors / actors' PII. Concerning anonymity, Waku v2 provides the following features:
+You can find more information on how to configure your application/service to use this in the language-specific documentation: [js-waku](https://github.com/waku-org/js-waku), [nim-waku](https://github.com/status-im/nwaku) and [go-waku](https://github.com/waku-org/go-waku)
 
-**Publisher-Message Unlinkability**:
-This feature signifies the unlinkability of a publisher to its published messages in the `WAKU-RELAY` protocol.
+## Predefined Bootstrap Nodes
 
-**Subscriber-Topic Unlinkability**:
-This feature stands for the unlinkability of the subscriber to its subscribed topics in the `WAKU-RELAY` protocol.
-The [Subscriber-Topic Unlinkability](https://rfc.vac.dev/spec/11/#security-analysis) is achieved through the utilization of a single PubSub topic.
-As such, subscribers are not re-identifiable from their subscribed topic IDs as the entire network is linked to the same topic ID.
-This level of unlinkability / anonymity is known as [k-anonymity](https://www.privitar.com/blog/k-anonymity-an-introduction/) where k is proportional to the system size (number of subscribers).
-Note that there is no hard limit on the number of the pubsub topics, however, the use of one topic is recommended for the sake of anonymity.
+Addresses of nodes hosted by Status are [predefined in the codebase](https://github.com/status-im/js-waku/blob/e4024d5c7246535ddab28a4262006915d2db58be/src/lib/discovery/predefined.ts#L48):
```

Comment:
```suggestion
An application using Waku can hardcode the addresses of bootstrap nodes directly in their codebase.
```

No need to assume it's Status nodes: they could have deployed their own node, etc.